utils/enums/example_page.py

```diff
@@ -4,7 +4,7 @@
 class ExamplePage(Enum):
     WITHOUT_SUBMIT_BUTTON = "noSubmitButton=true&"
     WITH_ADDITIONAL_BUTTON = "additionalButton=true&"
-    WITH_UPDATE_JWT = "updateJwt=ToDo&"
+    WITH_UPDATE_JWT = "updateJwt=%s&"
```

Comment:
Is this working correctly with updateJwt param? Probably it should be "update**d**Jwt="